contrib/ansible/group_vars/all.yml

```diff
@@ -6,7 +6,7 @@ cluster_name: cluster.local
 # Account name of remote user. Ansible will use this user account to ssh into
 # the managed machines. The user must be able to use sudo without asking
 # for password unless ansible_sudo_pass is set
-#ansible_ssh_user: root
+ansible_ssh_user: fedora
```

Comment:
@eparis I would say yes at this point since we are treating the group_vars as the 'per install' configuration file to go to. And I think its a good way to do it right now. 

As a side note on that you can just force add the file even if its in the gitignore. So not so bad to update.

@jayunit100 I don't have a problem moving some of them around but leaving the file with all of the comments explaining what is going on for each. on the other hand The way that the variables are inherited from one role to another will change if we move them around and that would call for some serious regression testing on all the platforms.